lib/application.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Application
+ * @constructor
+ * @param {Object} client Catapult client
+ */
+
+var Application = function (client) {
+	/**
+	 * List the user's applications
+	 * @param {Object} params Parameters for filtering applications
+	 * @param {Number} [params.page] The specified page requested when querying
```

Comment:
We do not want to allow people to use the page param. If someone needs the 2nd page, they should follow the 'next' link.